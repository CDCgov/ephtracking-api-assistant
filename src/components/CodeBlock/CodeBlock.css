.code-block {
  font-family: 'Source Code Pro', monospace;
  font-size: 85%;
  word-break: break-all;
  background: rgba(0, 0, 0, 0.08);
  padding: 11px 16px;
  line-height: 18px;
}