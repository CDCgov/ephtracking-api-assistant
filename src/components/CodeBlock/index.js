import React from 'react';
import PropTypes from 'prop-types';

import './CodeBlock.css';

const CodeBlock = ({ children }) => (
  <div className="code-block">{children}</div>
);

CodeBlock.propTypes = {
  children: PropTypes.node.isRequired
};

export default CodeBlock;