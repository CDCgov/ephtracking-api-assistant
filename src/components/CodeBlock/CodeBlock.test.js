import React from 'react';
import CodeBlock from '.';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<CodeBlock>Hello, World</CodeBlock>).toJSON();
  expect(tree).toMatchSnapshot();
});
