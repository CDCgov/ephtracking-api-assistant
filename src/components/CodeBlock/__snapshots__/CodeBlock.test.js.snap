// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="code-block"
>
  Hello, World
</div>
`;
